api/spree_api.gemspec

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |gem|
   gem.version       = version
 
   gem.add_dependency 'spree_core', version
+  gem.add_dependency 'active_model_serializers', '0.9.3'
```

Comment:
I believe we want to go with ~> 0.8.x here as 0.10.x is being based off 0.8 not 0.9.